### PR TITLE
Update versions for 0.26.0

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

File: pkg/kn/config/config_test.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func TestBootstrapConfig(t *testing.T) {
 	configYaml := `
 plugins:
   directory: /tmp
-  path-lookup: true
+
```
Comment (rhuss) [inline pkg/kn/config/config_test.go:32]: not sure why this slipped in, but should not harm (as this ensure that path-lookup is the default mode)

File: pkg/kn/config/config_test.go [hunk 2]
```diff
@@ -28,8 +28,7 @@ import (
 func TestBootstrapConfig(t *testing.T) {
 	configYaml := `
 plugins:
-  directory: /tmp
-  path-lookup: true
+  directory: /tmp
```
Comment (vyasgun) [inline pkg/kn/config/config_test.go:31]:
```suggestion
  directory: /tmp
```

**Comment by rhuss** [comment]
@vyasgun sorry, had to rebase. Could I get another `/lgtm` please ? :)

**Pull request merged**